evennia/typeclasses/models.py

```diff
@@ -832,6 +832,16 @@ def at_rename(self, oldname, newname):
         """
         pass
 
+    @classmethod
+    def get_concrete_content_type(cls):
```

Comment:
The 'concrete' part of this name is confusing since it's not a term we've used anywhere in Evennia otherwise (it's a Django-thing). It may be clearer to name this `get_dbmodel_content_type`. 

Also, this is not used anywhere ... so if you want to add this (in this otherwise unrelated PR) you'll need to add a docstring explaining why this is useful.